### PR TITLE
Adding new transactor pallet to integritee runtime & shell runtime

### PR DESCRIPTION
**Pull request by coax1d**
Title states it all(This is for slot handover process)

Other change:
* Convert the shell-chain-spec to a raw one because it can't handle new genesis fields otherwise.

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -12717,6 +12746,19 @@ dependencies = [
...
+source = "git+https://github.com/integritee-network/pallets.git?branch=coax1d-test-transactor-branch#f4f570373e560efc692574bd728367fd30654a04"
+dependencies = [
+ "common-primitives 0.1.0 (git+https://github.com/integritee-network/pallets.git?branch=coax1d-test-transactor-branch)",
+ "cumulus-primitives-core",
+ "frame-support",
+ "parity-scale-codec",
+ "sp-std",
+ "xcm",
+]
+
```
Comment (clangenb) [inline Cargo.lock:12761]: Nice, now we can be sure that the duplicate substrate deps from the subxt dev-deps don't end up in the other crates. :)

**Review by clangenb — COMMENTED** [review]
Looks great to me so far!

**Review by clangenb — COMMENTED** [review]
Nice, looks as I imagined it!

**Comment by coax1d** [comment]
once we merge this will change the toml file relating to `pallet-xcm-transactor` and `xcm-transactor-primitive` crates

**Comment by clangenb** [comment]
I have tried to connect to the real shell-kusama-lease2 chain, which did not work.
However, I reversed engineered the bootnode address, and could not get it to work with master either. So I guess this is safe to merge for now, and we will test this later.

**Review by clangenb — APPROVED** [review]

**Pull request merged**